megamek/src/megamek/client/ui/swing/FiringDisplay.java

```diff
@@ -281,11 +281,13 @@ public void performAction() {
 
                     @Override
                     public boolean shouldPerformAction() {
+                        // can get called outside of the firing phase, need to check 'isVisible' first in expression.
                         if (!clientgui.getClient().isMyTurn()
-                                || ce().getAlreadyTwisted()
-                                || clientgui.getBoardView().getChatterBoxActive()
                                 || !display.isVisible()
-                                || display.isIgnoringEvents()) {
+                                || display.isIgnoringEvents()
+                                || ce() == null
```

Comment:
If cd() returns null, this will exit out of the expression and wont evaluate the latter conditions in the expression.